.github/workflows/iroha2-dev-update_base_coverage.yml

```
@@ -1,19 +1,19 @@
-name: I2::Dev::Upload Coverage
+name: I2::Dev::Update Base Coverage
 
 on:
   pull_request:
     branches: [iroha2-dev]
     types: [closed]
 
 jobs:
-  upload_coverage:
+  update_with_artifact:
```

```suggestion
  with_artifact:
```

`i2::Dev::Update Base Coverage::update_with_artifact` => `i2::Dev::Update Base Coverage::with_artifact`

---

I'd like job names to be full sentences, unlike trait methods

---

This is an admirable desire, but 
1. Full sentences are not more readable, especially given that they're truncated. 
2. `update_with_artifact` isn't a full sentence. 
3. and it focuses on the wrong level of abstraction. I don't care whether it has an artifact or not, or whether it has the word **update** twice in it. 
4. This would make it inconsistent with other workflows which do not have full sentences as job names. 
5. Writing_full_sentences_with_lower_snake_case_isn't_very_convenient. Much easier to rely on programming knowledge and the fact that the namespace is always prepended.